client/src/Config.ts

```diff
@@ -21,32 +21,32 @@
  */
 export class Config {
 
-    public static readonly WEBSOCKET_END_POINT = "wss://localhost:9443/lsp/debug";
     public static readonly LANGUAGE_CLIENT_ID = "wso2isLanguageServer";
     public static readonly LANGUAGE_CLIENT_NAME = "Language Server for WSO2 IS";
     public static readonly PATH_OAUTH = "/oauth";
     public static readonly VSCODE_SP_REDIRECT_URL = "http://localhost:8010/oauth";
     public static readonly SUCCESS_HTML_NAME = "success.html";
     public static readonly DIAGRAM_HTML_NAME = "diagram.html";
     public static readonly AUTHENTICATION_HTML_NAME = "oAuth.html";
-    public static readonly PATH_APPLICATION_IMPORT: (iamUrl, tenant) => string =
-        (iamUrl, tenant) => `${iamUrl}/t/${tenant}/api/server/v1/applications/import`;
 
-    public static readonly PATH_GET_AUTH_CODE: (iamUrl, clientID, redirectUri, scope) => string =
-        (iamUrl, clientID, redirectUri, scope) => `${iamUrl}/oauth2/authorize?response_type=code&` +
-            `redirect_uri=${redirectUri}&client_id=${clientID}&scope="${scope}`;
 
-    public static readonly PATH_GET_SCRIPT_LIBRARY_BY_NAME: (iamUrl, tenant, scriptLibraryName) => string =
-        (iamUrl, tenant, scriptLibraryName) => `${iamUrl}/t/${tenant}/api/server/v1/` +
+    public static readonly PATH_APPLICATION_IMPORT: (baseUrl) => string =
```

Comment:
Shall we specify the type of the argument here? For example: `(baseUrl: string) => string =`